chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/SchedulerCertifiedProductSearchDetailsAsync.java

```diff
@@ -31,4 +31,12 @@ public Future<CertifiedProductSearchDetails> getCertifiedProductDetail(
         CertifiedProductSearchDetails dto = certifiedProductDetailsManager.getCertifiedProductDetails(id);
         return new AsyncResult<CertifiedProductSearchDetails>(dto);
     }
+
+    @Async("jobAsyncDataExecutor")
+    public Future<CertifiedProductSearchDetails> getCertifiedProductDetailSMALL(
```

Comment:
Could we rename this method? Maybe getCertifiedProductDetailsWithoutUnattestedCriteria ? Or maybe we can come up with something more generic than that, but less generic than "small". :)